lib/doc_template/tables/base.rb

```diff
@@ -20,10 +20,11 @@ def parse(fragment, *args)
         @options = args.extract_options!
 
         # get the table
-        table = fragment.at_xpath("table//*[contains(., '#{self.class::HEADER_LABEL}')]")
-        @table_exists = table.present?
+        table_key_cell = fragment.at_xpath("table//tr[1]/td[1][contains(., '#{self.class::HEADER_LABEL}')]")
+        @table_exists = table_key_cell.present?
         return self unless @table_exists
 
+        table = table_key_cell.ancestors.select { |n| n.name.casecmp('table').to_i.zero? }.first
```

Comment:
not critical, just a preference - imho using `table_key_cell.ancestors('table').first` is more clear and maybe faster here.

Reply:
Agree.